src/components/Results.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+
+class Results extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      filter: {},
+      rowsSelected: new Set()
+    };
+  }
+
+  orderData = (e) => {
+    if (!this.props.bookingResults) return;
+    let elementId = e.target.id;
+
+    if (this.state.filter[elementId]) {
```

Comment:
I don't believe this `if ()` is ever true as you don't seem to update the `filter` in the state anywhere else? can you double check?